internal/pkg/store/secretsmanager/secretsmanager.go

```diff
@@ -61,6 +61,15 @@ func (s *SecretsManager) CreateSecret(secretName, secretString string) (string,
 	return aws.StringValue(resp.ARN), nil
 }
 
+func (s *SecretsManager) DeleteSecret(secretName string) error {
+	_, err := s.secretsManager.DeleteSecret(&secretsmanager.DeleteSecretInput{
+		SecretId: aws.String(secretName),
+		// RecoveryWindowInDays *int64 TODO specify?
```

Comment:
Can we set [ForceDeleteWithoutRecovery](https://docs.aws.amazon.com/sdk-for-go/api/service/secretsmanager/#DeleteSecretInput) to true instead?

Reply:
Ah yes, that was the option mentioned earlier. Thoughts @kohidave ?

Reply:
Related: https://github.com/aws/amazon-ecs-cli-v2/issues/655